Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -1255,6 +1276,7 @@ auto B3IRGenerator::addElemDrop(unsigned elementIndex) -> PartialResult
 auto B3IRGenerator::addTableSize(unsigned tableIndex, ExpressionType& result) -> PartialResult
 {
     // FIXME: Emit this inline <https://bugs.webkit.org/show_bug.cgi?id=198506>.
+    emitPrepareWasmOperation(m_currentBlock);
```

Comment:
given these are sprinkled everywhere, I wonder if it's worth having a helper function like "callWasmOperation" that takes all the arguments for the CCallValue, and internally it calls emitPrepareWasmOperation. That way we don't need to remember to do this everywhere that we are doing a wasm operation call.

Reply:
something like:
```
template <typename ...Args>
Value* callWasmOperation(Args&&... args) { ... }
```

Reply:
Changed.